File: 02-Basic/JavaScript/04-primeNumberTillRange.js
function isPrimeNumber(number) {
    if (number <= 1) {
        return false; 
    }

    for (let i = 2; i <= number-1; i++) {
        if (number % i === 0) {
            return false; 
        }
    }

    return true; 
}

function printPrimesInRange(start, end) {
    for (let number = start; number <= end; number++) {
        if (isPrimeNumber(number)) {
            console.log(number);  
        }
    }
}


printPrimesInRange(10, 50);

File: 02-Basic/python/04-prime_number_till_range.py
def is_prime_number(number):
    if number <= 1:
        return False 

 
    for i in range(2, number- 1):
        if number % i == 0:
            return False 
    
    return True  

def print_primes_in_range(start, end):
    for number in range(start, end + 1):
        if is_prime_number(number):
            print(number)  
print_primes_in_range(10, 50)
